homeassistant/components/notify/rocketchat.py

```diff
@@ -0,0 +1,83 @@
+"""
+Rocket.Chat notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.rocketchat/
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_URL, CONF_USERNAME, CONF_PASSWORD)
+from homeassistant.components.notify import (
+    ATTR_DATA, PLATFORM_SCHEMA,
+    BaseNotificationService)
+from homeassistant.exceptions import (
+    PlatformNotReady)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['rocketchat-API==0.6.1']
+
+CONF_ROOM = 'room'
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_URL): cv.string,
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Required(CONF_ROOM): cv.string,
+})
+
+
+@asyncio.coroutine
+def async_get_service(hass, config, discovery_info=None):
+    """Return the notify service."""
+    username = config.get(CONF_USERNAME)
+    password = config.get(CONF_PASSWORD)
+
+    url = config.get(CONF_URL)
+    room = config.get(CONF_ROOM)
+
+    try:
+        return RocketChatNotificationService(url, username, password, room)
+    except PlatformNotReady:
+        _LOGGER.error("Unable to initialise Rocket.Chat service.")
```

Comment:
You don't need to log this error here. It will be logged by the caller of this function if it returns `None`.

Reply:
Done in 2f0472415.